docs/_pages/releases.md

```diff
@@ -14,6 +14,7 @@ sidebar:
 * Added the ability to exclude fields & properties marked as non-browsable in the code editor from structural equality equivalency comparisons - [#1807](https://github.com/fluentassertions/fluentassertions/pull/1807)
 
 ### Fixes
+* `EnumAssertions.Be` did not determine the caller name - [#1833](https://github.com/fluentassertions/fluentassertions/pull/1833)
```

Comment:
🔧 I think this should be 1835, right?

Reply:
Yes, my heuristic failed.
Since #1833 didn't exists I guessed this PR was going to have that number.